doc/conf.py

```diff
@@ -151,17 +162,6 @@
     html_baseurl = "https://pygmt.org/latest/"
 release = __version__
 
-requires_python = metadata("pygmt")["Requires-Python"]
-with pygmt.clib.Session() as lib:
-    requires_gmt = ">=" + lib.required_version
-
-# These enable substitutions using |variable| in the rst files
-rst_epilog = f"""
-.. |year| replace:: {year}
-.. |requires_python| replace:: {requires_python}
-.. |requires_gmt| replace:: {requires_gmt}
-"""
```

Comment:
Ok to remove. Looks like we only use `{year}` in the copyright line at L143 above.